config/rapidez/system.php

```diff
@@ -10,6 +10,9 @@
     // Elasticsearch prefix.
     'es_prefix' => env('ELASTICSEARCH_PREFIX', 'rapidez'),
 
+    // Get Magento url from Database
+    'get_magento_url_from_db' => env('GET_MAGENTO_URL_FROM_DATABASE', false),
```

Comment:
Maybe drop the "get"?
```suggestion
    'magento_url_from_db' => env('MAGENTO_URL_FROM_DATABASE', false),
```

And add it to the `.env.example`? + some more information in the comment / docs on this would be nice